### PR TITLE
B3 liked video page

### PR DESCRIPTION
**Pull request by sumitk-singh**
Created the new page for liked video.
-all of the liked video will get store here
-user can also remove it.

please review it.

### CODE REVIEW

File: src/pages/CreateNewPlaylist/CreateNewPlaylist.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+button{
+    background-color: var(--color-orange);
+    color: var(--color-light);
+}
+/* end of page style */
+
+
+/*---------------? */
```
Comment (zoyatanweer) [inline src/pages/CreateNewPlaylist/CreateNewPlaylist.css:42]: remove if not required

File: src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+//                     <label htmlFor="">Playlist Name</label>
+//                     <input type="text" />
+//                     <button>Create Playlist</button>
+//                 </div>
+//             </div>
+//         </>
+//     )
+// }
+
+// export default CreateNewPlaylist;
```
Comment (zoyatanweer) [inline src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx:18]: remove if not required

File: src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  const inputRef = useRef(null);
+  const playlistItems = Object.keys(playlist);
+  console.log("Window", playlistWindow);
+  return (
+    <>
+      <div className="playlist__add"></div>
+      <div className="playlist__card">
+        <div className="playlist__card__top">
+          <p>Add to Playlist</p>
+          {/* <FontAwesomeIcon */}
```
Comment (zoyatanweer) [inline src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx:42]: remove comments

File: src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            >
+              Create
+            </button>
+          </div>
+        ) : (
+          <div
+            className="playlist__card__bottom"
+            onClick={() => setNewPlaylistRoute((route) => !route)}
+          >
+            {/* <FontAwesomeIcon icon={faPlus} /> */}
```
Comment (zoyatanweer) [inline src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx:113]: remove comments

File: src/pages/MyPlaylist/MyPlaylist.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+            <div className='main-container-myplaylist-page'>
+
+                <section className='section-a'>
+                    <Sidebar />
+                </section>
+
+                <section className='section-b'>
+                    <h2>My Playlists 1<span className='float-right'></span></h2>
+
+                    {/* <section className='myplaylist-page-section-1'>
```
Comment (zoyatanweer) [inline src/pages/MyPlaylist/MyPlaylist.jsx:17]: remove commented code

**Review by zoyatanweer — APPROVED** [review]
- code looks good and well structured.
- remove the comments if not required.
- good to merge with suggested changes.

File: src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx (hunk 3 above)
Comment (sumitk-singh) [reply src/pages/CreateNewPlaylist/CreateNewPlaylist.jsx:42]: ok

File: src/pages/MyPlaylist/MyPlaylist.jsx (hunk 5 above)
Comment (sumitk-singh) [reply src/pages/MyPlaylist/MyPlaylist.jsx:17]: okay

File: src/contexts/Authcontext.jsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+            userToken: response.data.encodedToken,
+            activeUser: response.data.foundUser,
+          })
+        );
+        setUser(response.data.foundUser);
+        setToken(response.data.encodedToken);
+        navigate(from, { replace: true });
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (aiman-ansari) [inline src/contexts/Authcontext.jsx:41]: Suggestion: Remove console log

**Review by aiman-ansari — APPROVED** [review]
Good folder structure, variable naming in also declarative  .
Remove unnecessary comments and console log.

**Pull request merged**